lib/Entity/Video.php

```diff
@@ -397,6 +397,12 @@ public function setKeywords($keywords)
      */
     public function getStills()
     {
+        //sorting preview's images from smallest to biggest
+        usort($this->stills, function (array $item1, array $item2) {
+            if (isset($item1['dimension']['height']) && isset($item2['dimension']['height'])) {
+                return $item1['dimension']['height'] > $item2['dimension']['height'];
```

Comment:
"Sort callback function"s may return 1, 0, -1.
Your function will return true (1) and void (0).

Would you please add some functionality to avoid type casting and even returning -1 when item2 is greater than item1?